Resync WPT tests for media queries

rniwa opened the pull request:
#### daad17122377a2fcc1764a976000629f484456d8
<pre>
Resync WPT tests for media queries
<a href="https://bugs.webkit.org/show_bug.cgi?id=243646">https://bugs.webkit.org/show_bug.cgi?id=243646</a>

Reviewed by Tim Nguyen.

Updated web platform tests for CSS media queries as of 5c79743a93cfc4ded75d99338225531936da0c56.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/forced-colors-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/media-query-matches-in-iframe-expected.txt: Renamed
  from LayoutTests/fast/css/media-query-matches-in-iframe-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/media-query-matches-in-iframe.html: Renamed
  from LayoutTests/fast/css/media-query-matches-in-iframe.html.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/navigation-controls.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-contrast-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-data-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-motion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/relative-units-005-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/relative-units-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/matchmedia-utils.js:
(query_is_unknown):
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/test_media_queries-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/test_media_queries.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/253204@main">https://commits.webkit.org/253204@main</a>
</pre>

nt1m commented on LayoutTests/imported/w3c/resources/import-expectations.json:
```diff
@@ -27,6 +27,7 @@
     "css/cssom-view": "import", 
     "css/filter-effects": "import", 
     "css/geometry": "import", 
+    "css/mediaqueries": "import", 
```
This is already below as `web-platform-tests/css/mediaqueries`. Can you please undo this change?

(This file should probably get cleaned up, but that's a different story)

rniwa committed:
Resync WPT tests for media queries
https://bugs.webkit.org/show_bug.cgi?id=243646

Reviewed by Tim Nguyen.

Updated web platform tests for CSS media queries as of 5c79743a93cfc4ded75d99338225531936da0c56.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/forced-colors-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/media-query-matches-in-iframe-expected.txt: Renamed
  from LayoutTests/fast/css/media-query-matches-in-iframe-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/media-query-matches-in-iframe.html: Renamed
  from LayoutTests/fast/css/media-query-matches-in-iframe.html.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/navigation-controls.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-contrast-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-data-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-motion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/relative-units-005-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/relative-units-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/matchmedia-utils.js:
(query_is_unknown):
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/test_media_queries-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/test_media_queries.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: https://commits.webkit.org/253204@main
